grafana-plugin/src/containers/Rotations/Rotations.tsx

```diff
@@ -100,27 +101,25 @@ class _Rotations extends Component<RotationsProps, RotationsState> {
 
     return (
       <>
-        <div className={cx('root')}>
+        <div id="rotations" className={cx('root')}>
```

Comment:
Can we namepsace these ids to be part of grafana oncall only maybe? Or even better, could we instead access these elements through refs so that we don't use ids?

Reply:
If not refs, then I think we should have some global enum/object defined where we have all ids defined and we use it across the codebase. Sth like:
```
export const HTML_ID = {
  ROTATIONS: 'oncall-rotations',
  ...
} as const;
```